Software Engineer Big Data.md

```diff
@@ -27,6 +18,8 @@ Help our growing team of data scientists and engineers to unleash the potential
 - Non-religious about specific systems, frameworks, languages or tools
 - Can focus on deliverables and small increments but also not afraid of huge changes and humongous amounts of data
 - Proud to show your github.com account, research papers or other interesting achievements
+- Functional programming skills is a plus
```

Comment:
I do not consider functional programming or spark a personal trait, that should be described in "Your Profile" even if it is a plus.